examples/streaming-big-dataset/data_setup.sql

```diff
@@ -0,0 +1,28 @@
+DROP DATABASE IF EXISTS StreamTestDB;
```

Comment:
I think we cannot display this script in BBE site. @keizer619 can you please verify this?

Reply:
Yes, I also wanted to verify that. I suppose in that case we can point to the GitHub script location?